packages/webapp/webapp_tests.js

```diff
@@ -66,132 +68,149 @@ Tinytest.add("webapp - content-type header", function (test) {
              "application/javascript; charset=utf-8");
 });
 
-Tinytest.add("webapp - additional static javascript", function (test) {
-  var origInlineScriptsAllowed = WebAppInternals.inlineScriptsAllowed();
-
-  var staticFilesOpts = {
-    staticFiles: {},
-    clientDir: "/"
-  };
+Tinytest.addAsync("webapp - additional static javascript", function (test, onComplete) {
+  const run = async () => {
```

Comment:
I think you can just make the whole test function `async`, and call `onComplete` after the last `await`. I don't think `Tinytest` will do anything smart with the resulting `Promise` object, but that's fine.